app/i18n/en/index.ts

```diff
@@ -596,6 +596,9 @@ const en: BaseTranslation = {
     notConfirmed:
       "Payment has been sent\nbut is not confirmed yet\n\nYou can check the status\nof the payment in Transactions",
     note: "Note or label",
+    sendMax: "send max",
```

Comment:
Can lines 599 and 601 start off with a capitalized letter.

Reply:
Done